src/containers/WeeklyReports/index.js

```diff
@@ -0,0 +1,150 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+import { connect } from "react-redux";
+import { bindActionCreators } from "redux";
+import PropTypes from "prop-types";
+
+import { getWeeklyReports } from "../../actions";
+import {
+  ObjectNested,
+  getFiltersFromUrl,
```

Comment:
it's not necessary to importe this functions: `pushFiltersToUrl,` `getFiltersFormUrl`

Reply:
👍